client/extensions/woocommerce/app/products/product-update.js

```diff
@@ -116,6 +116,7 @@ class ProductUpdate extends React.Component {
 				<ProductHeader
 					site={ site }
 					product={ product }
+					viewEnabled={ true }
```

Comment:
You can just include `viewEnabled` as a prop without the true. It will default to true.